### PR TITLE
removed docker stuff from repo and referenced now separate docker pro…

### PR DESCRIPTION
**Pull request by Ragnaroek**
…ject

as we discussed by mail and in the issue :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,248 @@ This guide will show how Rust programs can be cross compiled for the Raspberry
 Pi using Cargo. These instructions may or may not work for your particular
 system, so you may have to adjust the procedure to fit your configuration.
 
-You can build the cross compiler and your rust project...
-* ...by hand on your host machine -- flexible but also more complex ([see MANUAL.md](MANUAL.md))
-* ...utilizing a docker image and running inside a docker container -- simple but a little less flexible ([see DOCKER.md](DOCKER.md))
+There exists also a docker container version of the Rust cross-compiler
+for the Raspberry: https://github.com/Ragnaroek/rust-on-raspberry-docker.
```
Comment (Ogeon) [inline README.md:8]: Would you mind just changing "There exists also..." to "There is also..."? It's otherwise great!

**Comment by Ogeon** [comment]
Nice! I think this will be good. Remember that this comes with a few unresolved issues, that will have to be redirected. Do you want to make issues in your repo and link to them from here, or do you prefer some other method?

**Comment by Ragnaroek** [comment]
Changed the commit.

I will copy the docker releated issues to the new repo and reference back.

**Comment by Ragnaroek** [comment]
I did go through all the existing issues and think they should all be closed?

**Comment by Ogeon** [comment]
Alright, I commented on one of them, but otherwise I agree.

**Pull request merged**